docs/releases/1.8.3.txt

```diff
@@ -92,3 +92,6 @@ Bugfixes
 
 * Fixed a regression in the ``unordered_list`` template filter on certain
   inputs (:ticket:`25031`).
+
+* Checked and documented that ``settings.SECRET_KEY`` is a valid unicode
```

Comment:
Per our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions), this won't be backported to 1.8, so you may remove the release note. I don't think a note for 1.9 is needed.